model/Hierarchy.php

```diff
@@ -375,11 +414,18 @@ public function markingClasses($numChildrenMethod="numChildren") {
 	 *
 	 * @param int  $id   ID of parent node
 	 * @param bool $open If this is true, mark the parent node as opened
+     * @param string $childrenMethod The name of the instance method to call to get the object's list of children
+     * @param string $numChildrenMethod The name of the instance method to call to count the object's children
+     * @param int $nodeCountThreshold
+     * @param array $filteredIds list of ids that will be used to filter list items (reduce the result set before processing)
 	 * @return bool
 	 */
-	public function markById($id, $open = false) {
+	public function markById($id, $open = false, $childrenMethod = "AllChildrenIncludingDeleted",
```

Comment:
Yes, because I want to be consistent with the rest of the code in the `Hierarchy` class. Please note that $childrenMethod is correctly set in the `TreeDropDown` and propagated to the rest ofthe functionality, so it's not an issue.

I agree that this default should be changed to `AllChildren` in all instances of the code, but this change is beyond scope of this PR, thus I don't think it should be included.